File: PY/2339.py
# BEE 2339 - Aviões de Papel - Python

def avioes(c, p, f):
    return 'S' if p >= c*f else 'N'


c, p, f = input().split()
aux = list(map(lambda x: int(x), [c, p, f]))
c, p, f = aux[0], aux[1], aux[2]
print(avioes(c, p, f))

File: C/2339.c
#include <stdio.h>

char avioes(int c, int p, int f)
{
    return (p >= c * f) ? 'S' : 'N';
}

int main()
{
    int c, p, f;

    scanf("%d %d %d", &c, &p, &f);
    printf("%c\n", avioes(c, p, f));
    return 0;
}